components/eventing-controller/pkg/backend/utils/eventmesh_utils.go

```diff
@@ -0,0 +1,205 @@
+package utils
+
+import (
+	"crypto/sha1" //nolint:gosec
+	"fmt"
+	"strings"
+
+	apigatewayv1beta1 "github.com/kyma-incubator/api-gateway/api/v1beta1"
+	eventingv1alpha2 "github.com/kyma-project/kyma/components/eventing-controller/api/v1alpha2"
+	"github.com/kyma-project/kyma/components/eventing-controller/pkg/ems/api/events/types"
+	"github.com/mitchellh/hashstructure/v2"
+	"github.com/pkg/errors"
+)
+
+// bebSubscriptionNameMapper maps a Kyma subscription to an ID that can be used on the BEB backend,
```

Comment:
naming

Reply:
lets keep it for now as it is used in all the tests for old BEB. Updating it will create conflicts with other PRs.

Reply:
I meant the name beb? If so we can leave this

Reply:
updated